WebApp/autoreduce_webapp/autoreduce_webapp/views.py

```diff
@@ -9,15 +9,19 @@
 """
 import os
 import sys
+
 from django.shortcuts import render
 from django.template import RequestContext
-# pylint: disable=no-member
 
 from utils.project.structure import get_project_root
+
+# pylint: disable=no-member
```

Comment:
I don't think we need that sys.path.append at all. I've removed a few before and it seemed to make no difference.

Does the import after it work without it? Note that this might also depend on where you start the webapp from:

- `python WebApp/autoreduce_webapp/manage.py runserver`, with cwd in the root of the repository
- `python manage.py runserver`, with cwd in `WebApp/autoreduce_webapp`

Reply:
Yeah there are quite a few of these. I was going to open a separate issue to look at this, but don't mind doing it here if the size is alright.